yarn.lock

```diff
@@ -1621,6 +1621,11 @@
   resolved "https://registry.yarnpkg.com/@fluentui/react-icons/-/react-icons-1.1.141.tgz#87e04a8f8302b7b8d0d81528a0596731dad2f3de"
   integrity sha512-9/wIK7/Tv6Ya5BHYgEONNN0KBJzcZhwvsCoV5ByFp+mNMc0mfdSi7VfOsgtWgUbORzIJdXqW7BF1gXA2hGkT1A==
 
+"@fluentui/react-icons@^2.0.150-beta.1":
```

Comment:
It seems that not all usages of `@fluentui/react-icons` were updated as there is still an entry for `@fluentui/react-icons@^1.1.140` in `yarn.lock`. Can you please check?

Reply:
I just took a look at this in case there was anything weird going on--when I ran `yarn` the duplicate went away.